src/backend/joanie/core/enums.py

```diff
@@ -79,3 +79,6 @@
         pgettext_lazy("As in: the enrollment failed on the LMS.", "Failed"),
     ),
 )
+
+# For Document issuers
+DOCUMENT_CONTRACT_DEFINITION = "contract_definition"
```

Comment:
What do you think to use this name ?

```suggestion
ISSUER_TEMPLATE_CONTRACT_DEFINITION = "contract_definition"
```

Reply:
It seems better, I fully agree. Especially for when we will have more than one template to manage.